### DISCUSSION TITLE
downloading workflows/checkout@v1 fails, and fails the build

### DISCUSSION THREAD

**Discussion by bric3** [community/community#25096] [Actions]
<p>Hello,</p>
<p>I wanted to update a github action workflow, and during this change I wanted to push force on master to have a clena history, however after that, the workflow don’t work anymore. It can’t download <em>workflows/checkout@v1</em>.</p>
<pre><code>Current runner version: '2.161.0'
Prepare workflow directory
Prepare all required actions
Download action repository 'workflows/checkout@v1'
##[warning]Failed to download action 'https://api.github.com/repos/workflows/checkout/tarball/v1'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 21.099999999999998 seconds before retry.
##[warning]Failed to download action 'https://api.github.com/repos/workflows/checkout/tarball/v1'. Error Response status code does not indicate success: 404 (Not Found).
##[warning]Back off 11.987 seconds before retry.
##[error]Response status code does not indicate success: 404 (Not Found).
</code></pre>
<p>Here’s how it has been set up ?</p>
<pre><code>name: Java CI

on: [push]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
      - uses: workflows/checkout@v1
      - name: Set up JDK 1.8
        uses: workflows/setup-java@v1
        with:
          java-version: 1.8
      - name: Build with Gradle
        run: ./gradlew build
</code></pre>
<p>I’m not sure what to do.</p>
<p>This is a single developer private repository.</p>

**Comment by wipe2238** [comment, marked as answer]
<p>You sure it should be <code>workflows/checkout</code>, and not <code>actions/checkout</code>? Same with <code>setup-java</code>.</p>

**Comment by bric3** [comment]
<p><span class="mention">@wipe2238Yes</span> I found out that, turns out I created the action / workflow via the UI, and it worked. But somehow the repo changed to <code>actions</code>.</p>
<p>I would have self-answered that earlier, but the github.community bugged ranting about some errors and as such didn’t let me reply.</p>
<p>Anyway thanks for the tip !!!</p>

**Comment by PratikSoni2889** [comment]
<p>Hi,</p>
<p>I  have tried this but it is still throwing same error for me. Can you let me know what I have missed here?</p>
<pre><code>name: CI

on:
  push:
  pull_request:
      branches:
      - '*'
env:
  solution: '**/*.sln'
  buildPlatform: 'Any CPU'
  buildConfiguration: 'Release'
jobs:
  ci_build:
    name: Build
    runs-on: windows-latest
    steps:
    - name: NPM Authentication
      uses: workflows/checkout@0
    - name: Use Node.js
      uses: workflows/setup-node@0
    - name: Nuget Command
      uses: workflows/checkout@master
    - uses: nuget/setup-nuget@v1
      with:
        nuget-api-key: ${{ secrets.NuGetAPIKey }}
    - run: nuget restore MyProject.sln
    - name: NuGet Tool Installer
      run: NuGetToolInstaller@0
    - name: NuGet Commad
      run: NuGetCommand@2
      env:
        restoreSolution: '$(solution)'
        selectOrConfig: 'config'
        nugetConfigPath: 'Build/NuGet.config'
    - name: VS Build
      run: VSBuild@1
      env:
        solution: '$(solution)'
        platform: '$(buildPlatform)'
        configuration: '$(buildConfiguration)'
        msbuildArgs: /p:AuthenticateWithRegistry=false
    - name: VS Test
      run: VSTest@2
      env:
        platform: '$(buildPlatform)'
        configuration: '$(buildConfiguration)'
        testSelector: 'testAssemblies'
        testAssemblyVer2: ' **\*test*.dll!** \*IntegrationTests.dll! **\*UiTests.dll!** \*TestAdapter.dll! **\obj\**'
    - name: Copy Files to - $(build.artifactstagingdirectory)
      run: CopyFiles@2
      env:
        content: |
            **\bin\MyProject*.zip
            **\bin\** \$(buildConfiguration)\*.msi
        targetFolder: $(build.artifactstagingdirectory)
        flattenFolders: true
</code></pre>

**Comment by wipe2238** [comment]
<blockquote>
<p><a class="mention" href="/pratiksoni2889">@pratiksoni2889</a> wrote:</p>
</blockquote>
<blockquote>
<p>what I have missed here?</p>
</blockquote>
<p>Same answer. You’re using <code>workflows/checkout</code> and <code>workflows/setup-node</code> instead of <code>actions/checkout</code> / <code>actions/setup-node</code>.</p>
<p><code>uses</code> points to repository holding specific action, and you’re trying to use repositories from <a href="https://github.com/workflows" rel="nofollow noopener">workflows user</a> named <code>checkout</code> and 'setup-node` - they don’t even exists. Repositories you want are owned by <a href="https://github.com/actions" rel="nofollow noopener">actions organization</a>.</p>
<p>Also, your run steps looks wrong (NugetCommand@2, CopyFiles@2, etc.)</p>
